cycode/cli/code_scanner.py

```diff
@@ -991,7 +991,9 @@ def _map_detections_per_file(detections) -> List[DetectionsPerFile]:
 def _get_file_name_from_detection(detection):
     if detection['category'] == "SAST":
         return detection['detection_details']['file_path']
-
+    if detection['category'] == "SecretDetection":
+        # get the full path by combining file_path and file_name
+        return f"{detection['detection_details']['file_path']}{detection['detection_details']['file_name']}"
```

Comment:
Extract to method - get_secret_detection_full_path and in the comment maybe add an example instead of explain the code below for better readibility.